spec/Secure Debug Proposal.md

```diff
@@ -92,9 +86,20 @@ Bits 8-11 of mdbgsec controls RISC-V trace
 | Yes | 1 | 1 | 0 | VU mode trace enabled |
 | Yes | 1 | 1 | 1 | VU/VS mode trace enabled |
 
-> 💡 Similar to mdbgsec.dbgprv/mdbgsec.dbgen, the default value of mdbgsec.trcprv/mdbgsec.trcen is implementation specific.
+> 💡 Similar to mdbgsec.dbgprv/mdbgsec.dbgen, the value of mdbgsec.trcprv/mdbgsec.trcen could be enforced by hardware fusing. 
+ 
+Bits 4-17 of mdbgsec:
+
+| Field          | Bit | Description                                                                                                          | Access         | Reset |
+| -------------- | --- | -------------------------------------------------------------------------------------------------------------------- | -------------- | ----- |
+| trclock        | 7   | 0: trcen, trcv, trcrprv are allowed to program <br> 1: trcen, trcv, trcprv values are locked and no more programable | Write 1 sticky | 0     |
+| dbglock        | 6   | 0: dbgen, dbgv, dbgprv are allowed to program <br> 1: dbgen, dbgv, dgbprv values are locked and no more programable  | Write 1 sticky | 0     |
+| extrigdis      | 5   | 0: external triggers are enabled <br>1: external triggers are disabled                                               | R/W            | 0     |
```

Comment:
Also sticky?

Reply:
I think it could be non-sticky and leave external trigger as machine-managed. They could be considered as kind of hardware peripherals owned by machine mode. Moreover, we have dbgprv to limit maximum debugable privilege level. Even the external trigger fires, the harts are still protected.